vendor.conf

```diff
@@ -103,7 +103,7 @@ github.com/googleapis/gax-go da06d194a00e19ce00d9011a13931c3f6f6887c7
 google.golang.org/genproto d80a6e20e776b0b17a324d0ba1ab50a39c8e8944
 
 # containerd
-github.com/containerd/containerd 89623f28b87a6004d4b785663257362d1658a729 # v1.0.0
+github.com/containerd/containerd 3fa104f843ec92328912e042b767d26825f202aa
```

Comment:
Should we add a note somewhere about this no longer being in sync with https://github.com/moby/moby/blob/2c8018f4bd7f48bf8f35770dea68f81b9591bb58/hack/dockerfile/binaries-commits#L7 ?

Reply:
Sure, where should I add the note? in the vendor.conf?

Reply:
Maybe in both; I don't know if there's still _some_ relation between the two?

I assume it's probably ok to make vendor.conf being ahead of the version of containerd we use, but when we bump containerd (e.g. go to v1.0.1 or v1.0.2), should we bump the vendored dependencies as well? (If there's no real need, the note could mention such, i.e. "consider bumping").

LOL, hope I make sense 😊

Reply:
Ok, I added a note to hack/dockerfile/binaries-commits

Updating vendor.conf does not require any other change so it doesn't seem necessary to document  "don't worry about it"

Reply:
We should ship the 1.0 binaries (or package them) but allow the vendored version to update to whatever docker wants to use.